app/controllers/events_controller.rb

```diff
@@ -11,5 +11,6 @@ def index
 
   def show
     @event = Event.find_by(slug: params[:id])
+    not_found unless @event
```

Comment:
As `not_found` raises `ActiveRecord::RecordNotFound` I think it'd be better and more "Rails-y" to just use `.find_by!` in Rails which raises `ActiveRecord::RecordNotFound` by itself: https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by-21

My suggestion is to simply do:

```ruby
@event = Event.find_by!(slug: params[:id])
```